activities/codebase-stewardship/supporting-governance/governance-template.md

```diff
@@ -9,89 +9,89 @@ The text below can be used as a template for creating a governance file in a rep
-* Maintain a technical roadmap, an architecture and coding principles
-* Resolve issues in development or conflicts between contributors
-* Managing and planning releases
-* Controlling rights to [codebase] assets such as source repositories
+Ideally, no one company or organization will employ a simple majority of the technical steering team.
 
-The technical steering team meets regularly. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly being done through the process described in CONTRIBUTING.md [add link].
+Any active member of the community can request to become a technical steering team member by asking the technical steering team. Members are appointed by the current technical steering team members.
 
-If more than one technical steering team member disagrees with a decision of the technical steering team they may appeal to the product steering team who will make the ultimate ruling.
+### Working process
 
-Ideally, no one company or organization will employ a simple majority of the technical steering team.
+The technical steering team meets regularly [explain frequency]. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly done through the process described in CONTRIBUTING.md [add link].
 
 ## Product steering team
 
-Responsibilities of the product steering team:
+The product steering team is responsible for:
 
-* Maintaining the mission, vision, values, and scope of the project
-* Collecting planned features and presenting them in a unified view
-* Refining the governance as needed
-* Making codebase level decisions
-* Resolving escalated project decisions when the sub-team responsible is blocked
-* Managing the [codebase] brand
-* Licensing and intellectual property changes
-* Controlling access to [codebase] assets such as hosting and project calendars
+* maintaining the mission, vision, values, and scope of the project
+* collecting planned features and presenting them in a unified view
+* refining the governance as needed
+* making codebase level decisions
+* resolving escalated project decisions when the sub-team responsible is blocked
+* managing the [codebase] brand
+* licensing and intellectual property changes
+* controlling access to [codebase] assets such as hosting and project calendars
 
-Members are appointed by the current product steering team members.
+Ideally, no one company or organization will employ a simple majority of the product steering team.
 
-Changes in codebase governance can only be done by the product steering team and requires super majority.
+Any active member of the community can request to become a product steering team member by asking the product steering team. Members are appointed by the current product steering team members.
```

Comment:
About the removal: now it is ambiguous which team can make changes to the codebase governance. 
About the addition: We talked about this process but thought it were better to have the product steering team just appoint members to avoid unnecessary voting. But this method has merits too.

Reply:
Re the removal: this information now sits under ## Decision making process.
Re the addition: my goal was to add information for community members about how new people came to join and how they could join - fundamentally, the process needs to feel fair.

Reply:
Regarding removal: it's not clear under the header Decision making process that the tech team don't have authority to do such changes, only the product team gets to do that.
Regarding addition: I'm fine with it, I just wanted to explain that it was not an omission in the first version, it was not added on purpose.